### PR TITLE
Fix personal blog soft filter

### PR DESCRIPTION
**Pull request by Discordius**
Due to a bug in the filtering code, if you only had a soft-filter on personal blog but no other tag, that wouldn't actually affect the result. Didn't show up in testing because I always had some other soft-filter enabled.

### CODE REVIEW

File: packages/lesswrong/lib/collections/posts/views.ts [hunk 1]
```diff
@@ -241,7 +244,7 @@ function filterSettingsToParams(filterSettings: FilterSettings): any {
   const frontpageFiltering = getFrontpageFilter(filterSettings)
   
   const {filter: frontpageFilter, softFilter: frontpageSoftFilter} = frontpageFiltering
-  
+  console.log(frontpageSoftFilter)
```
Comment (Raemon) [inline packages/lesswrong/lib/collections/posts/views.ts:247]: remove the console log

File: packages/lesswrong/lib/collections/posts/views.ts [hunk 2]
```diff
@@ -275,6 +278,8 @@ function filterSettingsToParams(filterSettings: FilterSettings): any {
       },
     };
   }
+
+  console.log(scoreExpr?.syntheticFields?.score)
```
Comment (Raemon) [inline packages/lesswrong/lib/collections/posts/views.ts:282]: same

**Comment by Discordius** [comment]
Done

**Review by Raemon — APPROVED** [review]

**Comment by Raemon** [comment]
Approved, conditional on "you tested it and it definitely works correctly"

**Pull request merged**